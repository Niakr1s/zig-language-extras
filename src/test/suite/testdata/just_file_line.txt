/home/ianic/zig/io_uring/loop/src/tcp.zig:218:29: 0x2237f3 in write (test)
        self.completion.args.send.buffer = buffer;
                            ^
/home/ianic/zig/io_uring/loop/src/tcp_echo_test.zig:186:30: 0x25ba5b in writeResolve (test)
            self.writer.write(self.writer_buffer[self.writer_pos..to]);
                             ^
/home/ianic/zig/io_uring/loop/src/tcp.zig:185:28: 0x249473 in complete (test)
                    resolve(ctx, no_bytes);
                           ^
/usr/local/zig/zig-linux-aarch64-0.11.0-dev.3886+0c1bfe271/lib/std/Thread.zig:433:13: 0x249b6b in callFn__anon_7160 (test)
            @call(.auto, f, args) catch |err| {
            ^

src/main.zig:33:19: error: error is ignored
src/main.zig:33:19: note: consider using 'try', 'catch', or 'if'

/usr/local/zig/zig-linux-aarch64-0.11.0-dev.3886+0c1bfe271/lib/std/start.zig:593:45: error: 'main' is not marked 'pub'
examples/spell_checker.zig:4:1: note: declared here
/usr/local/zig/zig-linux-aarch64-0.11.0-dev.3886+0c1bfe271/lib/std/start.zig:542:12: note: called from here
