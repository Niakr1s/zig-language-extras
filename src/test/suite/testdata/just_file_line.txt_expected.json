[
  {
    "file": "/home/ianic/zig/io_uring/loop/src/tcp.zig",
    "line": 217,
    "column": 28,
    "message": "in write (test)",
    "severity": 0
  },
  {
    "file": "/home/ianic/zig/io_uring/loop/src/tcp_echo_test.zig",
    "line": 185,
    "column": 29,
    "message": "in writeResolve (test)",
    "severity": 0
  },
  {
    "file": "/home/ianic/zig/io_uring/loop/src/tcp.zig",
    "line": 184,
    "column": 27,
    "message": "in complete (test)",
    "severity": 0
  },
  {
    "file": "/usr/local/zig/zig-linux-aarch64-0.11.0-dev.3886+0c1bfe271/lib/std/Thread.zig",
    "line": 432,
    "column": 12,
    "message": "in callFn__anon_7160 (test)",
    "severity": 0
  },
  {
    "file": "/project/root/src/main.zig",
    "line": 32,
    "column": 18,
    "message": "error is ignored",
    "severity": 0
  },
  {
    "file": "/project/root/src/main.zig",
    "line": 32,
    "column": 18,
    "message": "error is ignored\nconsider using 'try', 'catch', or 'if'",
    "severity": 2
  },
  {
    "file": "/usr/local/zig/zig-linux-aarch64-0.11.0-dev.3886+0c1bfe271/lib/std/start.zig",
    "line": 592,
    "column": 44,
    "message": "'main' is not marked 'pub'",
    "severity": 0
  },
  {
    "file": "/project/root/examples/spell_checker.zig",
    "line": 3,
    "column": 0,
    "message": "'main' is not marked 'pub'\ndeclared here",
    "severity": 2
  },
  {
    "file": "/usr/local/zig/zig-linux-aarch64-0.11.0-dev.3886+0c1bfe271/lib/std/start.zig",
    "line": 541,
    "column": 11,
    "message": "'main' is not marked 'pub'\ncalled from here",
    "severity": 2
  }
]